src/freenas/usr/local/lib/middlewared_truenas/plugins/failover.py

```diff
@@ -1722,13 +1753,21 @@ def sync_internal_ips(middleware, iface, carp1_skew, carp2_skew, internal_ip):
             middleware.logger.warn(f'Failed to run {verb}({service})', exc_info=True)
 
 
+async def ready_system_sync_keys(middleware):
+    await middleware.call('failover.call_remote', 'failover.sync_keys_with_remote_node')
```

Comment:
I think it may make more sense to do this in the event plugin, like it was done before?